index.js

```diff
@@ -409,14 +438,14 @@ let rcl = {
 
 process.on('SIGINT', () => {
   debug('Catch ctrl+c event and will cleanup and terminate.');
-  rcl.shutdown();
+  rcl.tryShutdown();
   process.exit(0);
```

Comment:
Personally, I still prefer if rclnodejs does not install any handlers, it stills shutdown "normally" on ctrl+c for me (in linux, maybe it's different for other OS).

If we are installing handlers, I think we should at least remove `process.exit(0)`, if we shutdown all contexts, node should exit automatically when it finishes all pending promises.

Reply:
We should change it in your PR